bin/tigmint-cut

```diff
@@ -125,11 +134,19 @@ def launchFindBreakpoints(bedfile, window, num_processes, partitioned_contigLeng
             if chrom not in breakpoints:
                 breakpoints[chrom] = []
             breakpoints[chrom].append(pos)
+        
+        while not attempted_corr_queue.empty():
+            attempted_corrections = attempted_corr_queue.get()
+            for _ in attempted_corrections:
```

Comment:
Just a question about the implementation - what was the rationale for inserting lists of length one `[1]` in the queue instead of just say `1` as an indicator flag for an attempted correction? If you did it that way, couldn't you just add one to `total_attempted_corr` each time you call `attempted_corr_queue.get()` instead of iterating over the list of length one?

Reply:
Ahh my bad, I was originally getting an error so I inserted a `[1]` because I was doing it the wrong way, but you're totally right! Just made that change, sorry I'm still new to the multiprocessing 🤦‍♀️

Reply:
No worries! :D Multiprocessing definitely takes some getting used to, especially since we don't do it too often